setup.py

```diff
@@ -60,11 +60,12 @@
     name="reana-db",
     version=version,
     description=__doc__,
+    license="MIT",
```

Comment:
@tiborsimko, addressing https://github.com/reanahub/reana-db/issues/77#issuecomment-669292448: When the package license is only present in classifiers `pip show` doesn't show it. For now the only workaround is to use the `license` parameter, however, the [official docs](https://packaging.python.org/specifications/core-metadata/?highlight=license#license) say that this field should only be used to declare custom licenses. More detailed information https://github.com/pypa/pip/issues/6677

Reply:
Hmm, so there is no good solution right now... Anyway, if adding `license=MIT` does not break anything apparent, only goes a bit against the spirit of official docs, we could perhaps have it everywhere while releasing later alpha versions...